hana/test/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file should not exist. The `test/cds.js` file has to add the `@cap-js/hana` credentials.